corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -2,6 +2,53 @@
 {% load report_tags %}
 {% load i18n %}
 
+{% block js-inline %}
+<script id="caseUploadsResource" type="text/plain">{% url 'case_importer_uploads' domain %}</script>
+<script>
+(function () {
+    var caseUploadResource = $('#caseUploadsResource').text();
```

Comment:
Probably definition should be in a separate file and binding should be on the page. That's the closest thing we've come to to a convention for this, so you're right I ought to adhere to it.